docs/Consuming.md

```diff
@@ -24,6 +25,38 @@ Known limitations:
 * UTF-8 encoded keys and values only. If data is encoded differently, it will not be pretty.
 * One consumer group is created per topic (may change in the future to just have one for the extension).
 
+### Kafka file
+
+Consumming can be done by creating a [.kafka](KafkaFile.md#kafkafile) file. Write simple consumer using the following format:
+
+```
+CONSUMER consumer-group-id
+topic: json-events
+partitions: 0
+from: 1
+```
+
+For actually consuming the topic, click on the `Start consumer` link above the `CONSUMER` line:
+
+![Start Consumer with Kafka file](assets/start-consumer-from-kafkafile.png)
+
+The CONSUMER block specifies : 
+
+ * the `consumer group id` which is declared after CONSUMER (optional).
+ * the `topic` id (required).
+ * the `from` offset which can takes `earliest`, `latest` or a `number` as value (optional).
+ * the `partitions` which is a number or a range, or a combinaison of ranges (optional). Example: 

```

Comment:
`partitions`: the partition number(s), or a partitions range, or a combinaison of partitions ranges *[optional]*. eg:

Reply:
fixed